ibm_db2/tests/conftest.py

```diff
@@ -0,0 +1,50 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from copy import deepcopy
+
+import ibm_db
+import pytest
+
+from datadog_checks.dev import WaitFor, docker_run, run_command
+from datadog_checks.ibm_db2 import IbmDb2Check
+from .common import COMPOSE_FILE, CONFIG
+
+
+class DbManager(object):
+    def __init__(self, config):
+        self.target, self.username, self.password = IbmDb2Check.get_connection_data(
+            config['db'], config['username'], config['password'], config['host'], config['port']
+        )
+        self.db_name = config['db']
+        self.conn = None
+
+    def initialize(self):
+        run_command(
+            (
+                'docker exec ibm_db2 su - db2inst1 -c '
+                '"db2 -c create db {} using codeset utf-8 territory us"'.format(self.db_name)
+            ),
+            check=True,
+        )
+        run_command('docker exec ibm_db2 su - db2inst1 -c "db2 -c update dbm cfg using HEALTH_MON on"', check=True)
+        run_command('docker exec ibm_db2 su - db2inst1 -c "db2 -c update dbm cfg using DFT_MON_STMT on"', check=True)
+        run_command('docker exec ibm_db2 su - db2inst1 -c "db2 -c update dbm cfg using DFT_MON_LOCK on"', check=True)
+        run_command('docker exec ibm_db2 su - db2inst1 -c "db2 -c update dbm cfg using DFT_MON_TABLE on"', check=True)
+        run_command('docker exec ibm_db2 su - db2inst1 -c "db2 -c update dbm cfg using DFT_MON_BUFPOOL on"', check=True)
+
+    def connect(self):
+        ibm_db.close(ibm_db.connect(self.target, self.username, self.password))
+
+
+@pytest.fixture(scope='session')
+def dd_environment():
```

Comment:
Getting this when running e2e :(

```
ddev env start ibm_db2 py27-11.1    
Environment variable DD_API_KEY does not exist; a well-formatted fake API key will be used instead. You can also set the API key by doing `ddev config set dd_api_key`.
Setting up environment `py27-11.1`... failed!
Stopping the environment...
py27-11.1 create: /Users/gregzussa/go/src/github.com/DataDog/integrations-core/ibm_db2/.tox/py27-11.1
py27-11.1 installdeps: -e../datadog_checks_base[deps], -rrequirements-dev.txt
py27-11.1 develop-inst: /Users/gregzussa/go/src/github.com/DataDog/integrations-core/ibm_db2
py27-11.1 installed: DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.,atomicwrites==1.3.0,attrs==19.1.0,certifi==2018.11.29,chardet==3.0.4,coverage==4.5.2,-e git+git@github.com:DataDog/integrations-core.git@1401164553ea9f8cc8850fdd30dc518d6e3b7e5e#egg=datadog_checks_base&subdirectory=datadog_checks_base,-e git+git@github.com:DataDog/integrations-core.git@1401164553ea9f8cc8850fdd30dc518d6e3b7e5e#egg=datadog_checks_dev&subdirectory=datadog_checks_dev,-e git+git@github.com:DataDog/integrations-core.git@1401164553ea9f8cc8850fdd30dc518d6e3b7e5e#egg=datadog_ibm_db2&subdirectory=ibm_db2,ddtrace==0.13.0,docutils==0.14,funcsigs==1.0.2,idna==2.7,mock==2.0.0,more-itertools==5.0.0,msgpack-python==0.5.6,pathlib2==2.3.3,pbr==5.1.3,pluggy==0.9.0,prometheus-client==0.3.0,protobuf==3.5.1,py==1.8.0,py-cpuinfo==4.0.0,pytest==4.3.0,pytest-benchmark==3.2.2,pytest-cov==2.6.1,pytest-mock==1.10.1,PyYAML==3.13,requests==2.20.1,scandir==1.9.0,simplejson==3.6.5,six==1.11.0,statistics==1.0.3.5,uptime==3.0.1,urllib3==1.24.1,uuid==1.30,wrapt==1.11.1
py27-11.1 run-test-pre: PYTHONHASHSEED='3917960927'
py27-11.1 runtests: commands[0] | pip install -r requirements.in
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Collecting ibm_db==2.0.9 (from -r requirements.in (line 1))
  Using cached https://files.pythonhosted.org/packages/3f/61/389c6decacfed61c2724005b8c2be61f2c699f731b53af13517e5a0e6e98/ibm_db-2.0.9.tar.gz
Building wheels for collected packages: ibm-db
  Building wheel for ibm-db failed: [Errno 13] Permission denied: '/Users/gregzussa/Library/Caches/pip/wheels/ef/89'
Failed to build ibm-db
Installing collected packages: ibm-db
  Running setup.py install for ibm-db: started
    Running setup.py install for ibm-db: finished with status 'done'
Successfully installed ibm-db-2.0.9
py27-11.1 runtests: commands[1] | pytest -v --benchmark-skip
ImportError while loading conftest '/Users/gregzussa/go/src/github.com/DataDog/integrations-core/ibm_db2/tests/conftest.py'.
.tox/py27-11.1/lib/python2.7/site-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
tests/conftest.py:6: in <module>
    import ibm_db
E   ImportError: dlopen(/Users/gregzussa/go/src/github.com/DataDog/integrations-core/ibm_db2/.tox/py27-11.1/lib/python2.7/site-packages/ibm_db.so, 2): Library not loaded: libdb2.dylib
E     Referenced from: /Users/gregzussa/go/src/github.com/DataDog/integrations-core/ibm_db2/.tox/py27-11.1/lib/python2.7/site-packages/ibm_db.so
E     Reason: image not found
ERROR: InvocationError for command '/Users/gregzussa/go/src/github.com/DataDog/integrations-core/ibm_db2/.tox/py27-11.1/bin/pytest -v --benchmark-skip' (exited with code 4)
___________________________________ summary ____________________________________
ERROR:   py27-11.1: commands failed


Unable to parse configuration. Try recreating your env to get the latest version of the dev package.
```

Reply:
https://github.com/ibmdb/python-ibmdb/tree/master/IBM_DB/ibm_db#issues-with-mac-os-x ?